front/src/common/Loaders/Loader.tsx

```diff
@@ -22,13 +22,15 @@ export const Spinner: FC<SpinnerProps> = ({ displayDelay, ...props }) => {
     };
   }, [displayDelay, timeoutRef]);
 
-  return display ? (
-    <div {...props}>
-      <div className="spinner-border" role="status">
-        <span className="sr-only">Loading...</span>
+  return (
+    display && (
```

Comment:
return early

Reply:
I used a ternary here because the component is very small, done